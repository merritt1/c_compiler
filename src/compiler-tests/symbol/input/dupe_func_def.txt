int myFunc(void) {
    return 123;
}
int myFunc(void) {
    return 456;
}
int main(void) {
    return myFunc();
}
