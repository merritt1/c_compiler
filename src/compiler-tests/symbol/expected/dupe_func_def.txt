


***ERROR: Redefinition of identifier 'myFunc'


=============== PARSE TREE ===============
signed int (myFunc(void)) {
return 123;
}
signed int (myFunc(void)) {
return 456;
}
signed int (main(void)) {
return (myFunc());
}
================= SYMBOLS ================
Symbol Table ID: 0
  myFunc -> function(signed int, [void])
  main -> function(signed int, [void])

